README.md

```diff
@@ -1,7 +1,12 @@
 # Google Software Product Sprint
 
+<<<<<<< HEAD
```

Comment:
This "<<<<< HEAD" thing is the marker git puts for indicating merge conflicts, you should resolve the merge conflict and delete this marker afterwards.